.github/workflows/test.yml

```diff
@@ -81,9 +81,9 @@ jobs:
       run: npm run build
     - name: Test experimental features
       env:
-        ASC_FEATURES: threads,reference-types,gc,exception-handling
+        ASC_FEATURES: "*"
```

Comment:
Idea here was that we can add features that we know should work on Node canary, or disable them again if something breaks. Given that there is no `--disable` here, perhaps keeping a long list works better, but make sure we update the list more often?

Reply:
In the past, what would happen was that we implemented milestone X, and things worked with it, but not with milestone X+1. So during development, it was useful to run specific feature tests locally, but not run them on CI just yet, for example.

Reply:
> In the past, what would happen was that we implemented milestone X, and things worked with it, but not with milestone X+1. So during development, it was useful to run specific feature tests locally, but not run them on CI just yet, for example.

I see why you want to whitelist the features...Is it perhaps better if the regular test jobs compiled every test to match against the fixtures but only this job instantiated the tests, and only according to ASC_FEATURES?

Reply:
Perhaps, while prompting us to at least keep Binaryen and early feature tests in sync. What might happen is that we implemented something that Binaryen does't yet have interpreter support for, but not sure it that's relevant as it wouldn't work in manual local testing either and would rather be commented out in the test case itself.